### PR TITLE
Added correct link to fluidproject.org

### PR DESCRIPTION
**Pull request by Ayush23Dash**
The link to fluidproject.org in the current version does not point to the actual website where it is intended to. Making slight changes to the link does the work.

### CODE REVIEW

File: src/documents/news/2007-05-04-Better_Web_Experiences.html.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ The project will also foster the contributions of skilled interaction designers,
 
 Although Fluid will initially focus on improving academic software, the technology may also be useful in other web software products. Rich Schwerdtfeger, chair of the IBM Accessibility Architecture Review Board says, "Fluid will be the first project to deliver a personalized Web 2.0 experience for people with disabilities. Although the Fluid Project is targeted for the learning space, what we learn should have broader web implications in the future."
 
-Many of the Fluid Project partners are founding contributors to the academic community source software movement, and bring a wealth of design and development expertise to the project. Please visit [http://fluidproject.org] for more information or to get involved.
+Many of the Fluid Project partners are founding contributors to the academic community source software movement, and bring a wealth of design and development expertise to the project. Please visit [http://fluidproject.org ] for more information or to get involved.
```
Comment (Raunaksingh100) [inline src/documents/news/2007-05-04-Better_Web_Experiences.html.md:18]: no need of One Extra space   ,plz revert the changes ,
thanks..

**Review by Raunaksingh100 — COMMENTED** [review]
you just added the space in it ,
the link is already correct ....

File: src/documents/news/2007-05-04-Better_Web_Experiences.html.md (hunk 1 above)
Comment (Ayush23Dash) [reply src/documents/news/2007-05-04-Better_Web_Experiences.html.md:18]: > no need of One Extra space ,plz revert the changes ,
> thanks..

No actually the link was not working initially, hence the space was required!
